PerfDataExtension/pluginManifest.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/microsoft/microsoft-performance-toolkit-sdk/main/src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/Manifest/PluginManifestSchema.json",
+  "identity": {
+    "id": "Microsoft.Performance.Toolkit.Plugins.PerfDataExtension",
+    "version": "0.1.2"
+  },
+  "displayName": "Linux - Perf",
+  "description": "Enables viewing Linux tracepoint events recorded with the perf tool",

```

Comment:
While the support is tracepoint only for now, I recommend removing the tracepoint description here; as likely the support will change in the future to be more than tracepoints...